crypto/keys/multisig/amino.go

```diff
@@ -10,7 +10,7 @@ import (
 // Amino JSON marshaling of LegacyAminoPubKey (see below for details).
 //
 // This struct is copy-pasted from:
-// https://github.com/tendermint/tendermint/blob/v0.33.9/crypto/multisig/threshold_pubkey.go
+// https://github.com/cometbft/cometbft/blob/v0.33.9/crypto/multisig/threshold_pubkey.go
```

Comment:
this link doesn't exist, we should leave it tendermint since it was coped from there

Reply:
Whoops, changed 👍🏾